pkg/minishift/util/iso_path.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
+	"net/url"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+func GetIsoPath(isoURL string) string {
+	uri, _ := url.Parse(isoURL)
```

Comment:
So, you ignore an error, because you know in this context that the URL is valid? So this method depends on the caller making sure that the URL is valid first?

Reply:
If I'd call this method with an invalid URL, I would get a _nil_ uri and a potential panic?

Reply:
@hferentschik this method is called from the cluster package and the ISO URI comes from start.go (where we already check for URI validity). I purposefully ignore error here because of there already we check the validity of URI.

Reply:
maybe we should add a comment. Not that this method ever gets used from another context.  After all, it is in the util package. 

If it is so specific to this particular usecase maybe it should be a package private method there, instead of a public util method.

The alternative is to check the error and either return it or treat it is a default 'unamed' case.

Reply:
> maybe we should add a comment. Not that this method ever gets used from another context. After all, it is in the util package.

@hferentschik done.

> If it is so specific to this particular use-case maybe it should be a package private method there, instead of a public util method.

Right now this is used for cluster package so if I make it private to `util` then not able to use for cluster.

> The alternative is to check the error and either return it or treat it is a default 'unamed' case.

Yes, that's best bet, but I was checking what could be error from url.Parse since it accept raw strings https://play.golang.org/p/KuI1HLqcrC not able to get what kind of parse error we can have it here.